test/koa-mysql/app/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "koa-mysql-example",
+  "private": true,
+  "version": "1.0.0",
+  "main": "dist/app.js",
+  "scripts": {
+    "build": "tsc --version; tsc -p tsconfig.json",
```

```suggestion
    "build": "tsc -p tsconfig.json",
```
Is the version number being printed required?

---

It's not required per se, but I think it helps, knowing that a minimum version upgrade can break the build process.